### PR TITLE
Added support for getting cluster nodes details and migration status.

### PR DESCRIPTION
**Pull request by sharadkesarwani**
Added support for getting cluster nodes details and migration status.

# Jira Ticket
https://spotinst.atlassian.net/browse/SPOTAUT-17383

### CODE REVIEW

File: service/ocean/providers/aws/clusternodes.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+}
+type ReadClusterNodeInput struct {
+	ClusterID *string `json:"clusterId,omitempty"`
+}
+
+type ReadClusterNodeOutput struct {
+	ClusterNode []*ClusterNodes `json:"clusterNode,omitempty"`
+}
+
+func (s *ServiceOp) ReadClusterNodes(ctx context.Context, input *ReadClusterNodeInput) (*ReadClusterNodeOutput, error) {
```
Comment (chandra1-n) [inline service/ocean/providers/aws/clusternodes.go:61]: query parameters are not handled

File: service/ocean/providers/aws/migrationsummary.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
+package aws
```
Comment (chandra1-n) [inline service/ocean/providers/aws/migrationsummary.go:1]: please add get migration status as well, if it's not implemented already

**Review by chandra1-n — CHANGES_REQUESTED** [review]
please fix the review comments

File: service/ocean/providers/aws/clusternodes.go (hunk 1 above)
Comment (sharadkesarwani) [reply service/ocean/providers/aws/clusternodes.go:63]: Added usage of query param.

File: service/ocean/providers/aws/migrationsummary.go (hunk 2 above)
Comment (sharadkesarwani) [reply service/ocean/providers/aws/migrationsummary.go:1]: Added implementation for get migration status as well.

**Review by chandra1-n — APPROVED** [review]

**Pull request merged**